### PR TITLE
feat(TableRow): Table Row component implementation

### PR DESCRIPTION
**Pull request by masoudmanson**
## Summary

**Row**
Shortcut ticket: [sh-208408](https://app.shortcut.com/sci-design-system/story/208408/row-component)

Implemented with &lt;tr&gt; under the hood.

## Checklist

- [x] Default Story in Storybook
- [ ] LivePreview Story in Storybook
- [x] Test Story in Storybook
- [x] Tests written
- [x] Variables from `defaultTheme.ts` used wherever possible
- [ ] If updating an existing component, depreciate flag has been used where necessary
- [ ] Chromatic build verified by @chanzuckerberg/sds-design

### CODE REVIEW

**Review by clarsen-czi — CHANGES_REQUESTED** [review]
Just a small change regarding how row hovering is executed, but otherwise looking good!

File: src/core/Row/index.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React, { forwardRef } from "react";
+import Tooltip, { TooltipProps } from "../Tooltip";
+
```
Comment (mmmmmmaya) [inline src/core/Row/index.tsx:3]:
```suggestion
```

**Review by mmmmmmaya — APPROVED** [review]

File: src/core/Row/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    expect(elements).toBeTruthy();
+  });
+
+  it("renders tooltip on hover", async () => {
+    render(<Test />);
+    const rowElement = screen.getByTestId("TableRow");
+    fireEvent.mouseOver(rowElement);
+    await screen.findByText("testTooltipTitle");
+  });
+});
```
Comment (bethbertozzi) [inline src/core/Row/index.test.tsx:26]: could increase test coverage to include rendering of other props, like hover

**Review by clarsen-czi — COMMENTED** [review]
LGTM! Thanks @masoudmanson !

**Pull request merged**